.semaphore/semaphore.yml

```diff
@@ -56,55 +56,74 @@ blocks:
     task:
       secrets:
         - name: agent-aws-stack-tester-credentials
+      prologue:
+        commands:
+          - checkout
+          - cache restore venv-$SEMAPHORE_GIT_BRANCH-$(checksum requirements.txt)
+          - sudo apt-get update && sudo apt-get -y install python3-venv
+          - curl -sL https://releases.hashicorp.com/packer/1.7.9/packer_1.7.9_linux_amd64.zip -o /tmp/packer_1.7.9_linux_amd64.zip
+          - unzip /tmp/packer_1.7.9_linux_amd64.zip -d /tmp/
+          - sudo mv /tmp/packer /usr/local/bin/
       jobs:
-        - name: Build AMI
+        - name: Build Linux AMI
           commands:
-            - checkout
-            - cache restore venv-$SEMAPHORE_GIT_BRANCH-$(checksum requirements.txt)
-            - sudo apt-get update && sudo apt-get -y install python3-venv
-            - curl -sL https://releases.hashicorp.com/packer/1.7.9/packer_1.7.9_linux_amd64.zip -o /tmp/packer_1.7.9_linux_amd64.zip
-            - unzip /tmp/packer_1.7.9_linux_amd64.zip -d /tmp/
-            - sudo mv /tmp/packer /usr/local/bin/
             - ./ci/build-ami.sh ubuntu-focal amd64-server
```

Comment:
`amd64-server` is misleading as an architecture (arch), at least for me.

Reply:
I was using that value due to how the base Ubuntu images were named:

```
"ubuntu/images/*ubuntu-focal-20.04-amd64-server-*"
"ubuntu/images/*ubuntu-focal-20.04-arm64-server-*"
```

But you're right. I'll change it to use the `architecture` AMI filter instead (i386 | x86_64 | arm64)